doc/rados/operations/health-checks.rst

```diff
@@ -863,40 +866,45 @@ this may be done surgically for individual OSDs or a given mask
 WAL_DEVICE_STALLED_READ_ALERT
 _____________________________
 
-A similar warning like ``BLOCK_DEVICE_STALLED_READ_ALERT`` will be raised to
-identify ``stalled read`` instances on a given BlueStore OSD's ``WAL_DEVICE``.
-This warning can be configured via ``bdev_stalled_read_warn_lifetime`` and
-``bdev_stalled_read_warn_threshold`` parameters similarly described in the
+The warning state ``WAL_DEVICE_STALLED_READ_ALERT`` will be raised to
```

Comment:
s/will be/is/

I think that the present tense is easier to read than the future tense (except in certain finicky situations in which you're explaining a series of things that happen one after another, which this is not).

I think that "will be" should be changed to "is" throughout this document wherever doing so doesn't mangle the intended sense of the sentence.